plugin.go

```diff
@@ -15,20 +15,23 @@ type (
 	}
 
 	Build struct {
-		Tag      string
-		Event    string
-		Number   int
-		Commit   string
-		Ref      string
-		Branch   string
-		Author   string
-		Pull     string
-		Message  string
-		DeployTo string
-		Status   string
-		Link     string
-		Started  int64
-		Created  int64
+		Tag          string
+		Event        string
+		Number       int
+		Commit       string
+		Ref          string
+		Branch       string
+		Author       string
```

Comment:
IMHO it would be better to use a struct like https://github.com/drone/drone-plugin-starter/blob/master/plugin.go#L35-L39. To keep it backward compatible the struct could implement a `String` function which just prints the author name